### ISSUE TITLE
Can not find icon

### ISSUE THREAD

**Issue by ulongx** [flutter/flutter#12799]
![image](https://user-images.githubusercontent.com/1161624/32206203-bbd3fd6e-bdc1-11e7-8ca8-4bddb2fc8481.png)
``` dart
class TutorialHome extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return new Scaffold( 
      appBar: new AppBar( 
        leading: new IconButton(
          icon: new Icon(Icons.menu), tooltip: 'ddd', onPressed: null,),
        title: new Text('Tutorial Style'),
        actions: <Widget>[
          new IconButton(icon: new Icon(Icons.search), onPressed: null)],
      ),
      body: new Center(child: new Text('Hello World'),),
      floatingActionButton: new FloatingActionButton(
          tooltip: 'add', child: new Icon(Icons.add), onPressed: null),
    );
  }
}
```

**Comment by pedia** [comment]
me too since upgrade flutter yesterday.


[✓] Flutter (on Mac OS X 10.12.6 16G29, locale en-HK, channel master)
    • Flutter at /Users/odd/wa/flutter/flutter
    • Framework revision **50db9478e3** (19 hours ago), 2017-10-30 11:12:12 +0100
    • Engine revision ec19da1c80
    • Tools Dart version 1.25.0-dev.11.0

[✓] Android toolchain - develop for Android devices (Android SDK 26.0.2)
    • Android SDK at /Users/odd/Library/Android/sdk
    • Platform android-26, build-tools 26.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] iOS toolchain - develop for iOS devices (Xcode 9.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 9.0.1, Build version 9A1004
    • ios-deploy 1.9.2
    • CocoaPods version 1.3.1

[✓] Android Studio (version 2.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] Connected devices
    • iPhone 6s • 1F14B19C-80E9-4D83-8789-8DB4294BF5B4 • ios • iOS 11.0 (simulator)

**Comment by pedia** [comment]
And failed:

> 'package:flutter/src/rendering/box.dart': Failed assertion: line 1433 pos 12: 'hasSize': is not true.
> #0      _AssertionError._doThrowNew (dart:core-patch/dart:core/errors_patch.dart:37)
> #1      _AssertionError._throwNew (dart:core-patch/dart:core/errors_patch.dart:33)
> #2      RenderBox.size (package:flutter/src/rendering/box.dart:1433:12)
> #3      RenderProxyBoxWithHitTestBehavior.hitTest (package:flutter/src/rendering/proxy_box.dart:162:9)
> #4      RenderBox&ContainerRenderObjectMixin&RenderBoxContainerDefaultsMixin.defaultHitTestChildren (package:flutter/src/rendering/box.dart:2166:17)
> #5      RenderCustomMultiChildLayoutBox.hitTestChildren (package:flutter/src/rendering/custom_layout.dart:348:12)
> #6      RenderBox.hitTest (package:flutter/src/rendering/box.dart:1849:11)
> #7      RenderBox&RenderObjectWithChildMixin&RenderProxyBoxMixin.hitTestChildren (package:flutter/src/rendering/proxy_box.dart:116:19)
> #8      RenderBox.hitTest (package:flutter/src<…>

**Comment by eseidelGoogle** [comment]
I believe this is https://github.com/flutter/flutter/issues/12739 and is just waiting for us to roll the engine.

**Comment by eseidelGoogle** [comment]
We believe this to be fixed if you `flutter upgrade`.  Sorry for the trouble.

**Comment by aam** [comment]
@pedia, the fix was in framework rev 2e97ee4, didn't make it in 50db947 you looked at yet.

**Comment by ulongx** [comment]
> Locally, you should be able to edit your bin/internal/engine.version file to reference the engine commit - 7c4142808c700f939be5305067b94097894b3d11 - and then do something like run flutter precache.

fixed  :)  thanks

**Comment by PercyMcfly** [comment]
Having the same issue. 
The icons all work perfectly, usually. But after using the app for a while, it seems like no icon is being recognized anymore. I noticed the problem last week, and just upgraded Flutter today.. 
I only noticed it on iOS devices, mostly because i don‘t own an android device and don‘t run simulations for hours.. 
Currently experiencing the Issue in my iPhone 11, app had been running in background for about 5 hours. 
Also experienced the issues on an iPhone 7.
